changelog/generated/v1.22.1.md

```diff
@@ -0,0 +1,7 @@
+## v1.22.1
+
+### Bug Fixes
```

Comment:
Not able to see the changelog for `skip-tls-verify` and `use-http`

Reply:
@laxmikantbpandhare because it wasn't cherry picked to the release branch. :) We needed to add a cherry pick comment to the PR so that the bot would do it. I will cherrypick the commit manually and update this PR with the change.

Reply:
Ah! I was not aware of it. thanks for the confirmation. PR looks good to me.